db/silkworm/db/tables.hpp

```diff
@@ -29,7 +29,14 @@ namespace silkworm::db::table {
 constexpr lmdb::TableConfig kMAIN_DBI{nullptr};
 constexpr lmdb::TableConfig kAccountHistory{"hAT"};
 constexpr lmdb::TableConfig kBlockBodies{"b"};
-constexpr lmdb::TableConfig kBlockHeaders{"h"};
+
+constexpr lmdb::TableConfig kCanonicalHashes{"canonical_headers", 0,
```

Comment:
These new tables don't use a custom key comparator in TG. Why should they in Silkworm?

Reply:
Cause it improves speed.
Default lexical comparator takes into account length of compared keys: but we know these tables have fixed key len so there is no point in checking both lengths (the custom comparator does only a memcmp).
Checked on bulk loading 12M canonical hashes with default comparator from LMDB 19.8 seconds while on custom comparator 19.5 seconds (average on 20 rounds). Is not that much but nevertheless a gain and does not break compatibility amongs tg/silkworm

Reply:
OK, fair enough.